tests/e2e/e2e_test.go

```diff
@@ -12,44 +10,152 @@ import (
 	"github.com/umee-network/umee/v4/tests/grpc"
 )
 
+func (s *IntegrationTestSuite) checkOutflows(umeeAPIEndpoint, denom string, checkWithExcRate bool, amount sdk.Dec, excDenom string) {
+	s.Require().Eventually(
+		func() bool {
+			outflows, err := queryOutflows(umeeAPIEndpoint, denom)
+			s.Require().NoError(err)
+			if checkWithExcRate {
+				outflow := outflows.AmountOf(denom)
+				// get exchange rate
+				exchangeRate, err := queryExchangeRate(umeeAPIEndpoint, excDenom)
+				s.Require().NoError(err)
+				powerReduction := sdk.MustNewDecFromStr("10").Power(6)
+				totalPrice := amount.Quo(powerReduction).Mul(exchangeRate.AmountOf(excDenom))
+				s.T().Log("exchangeRate total price ", totalPrice.String(), "outflow value", outflow.String())
+				oncePer := totalPrice.Quo(sdk.MustNewDecFromStr("100"))
+				threePen := oncePer.Mul(sdk.MustNewDecFromStr("3"))
```

Comment:
why do you divide by 100 and multiply by 3?

Reply:
maybe this should be a function parameter? If this requires explanation, then add a comment about that arguments.